packages/framework/src/Component/Error/ExceptionListener.php

```diff
@@ -1,38 +0,0 @@
-<?php
-
-declare(strict_types=1);
-
-namespace Shopsys\FrameworkBundle\Component\Error;
-
-use Symfony\Component\HttpKernel\Event\ExceptionEvent;
-use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
-use Throwable;
-use Twig\Error\RuntimeError;
-
-class ExceptionListener
```

Comment:
please do not remove this exception listener. HttpNotFound during subrequest is properly rendered as a 404 page instead of error thanks to this.

Reply:
ok :( fixed